Cli.py

```diff
@@ -66,6 +68,8 @@ def session(self):
         course_id_record=self.cursor_1.fetchall()
         for i in course_id_record:
             self.course_id=i[0]
+                #list comp maybe
+
         self.session_dict["course_id"]=self.course_id
```

Comment:
Some of your `session_dict` and other dictionary additions could possibly be consolidated into a function that takes user input, and if it gets an invalid format, can tell the user such and then try again without stopping the whole CLI. I noticed this in line ~46 that a try/except would just skip goals and notes.

This function could be general and take arguments like a `**kwargs` where each key is the key in the dict you want to return results for, and the value might be a tuple of (prompt, type_expected) like `("Did you have notes? ", str)`. We can talk about this more on the next call

Reply:
BTW I don't mean the `session()` method when I say "function"